### ISSUE TITLE
Custom --config-dir chokes on copying options-ssl.conf

### ISSUE THREAD

**Issue by kuba** [letsencrypt/letsencrypt#451]
```
(venv)root@le:~/lets-encrypt-preview# rm -rf /etc/letsencrypt/ /var/lib/letsencrypt/ && letsencrypt -vv -ted foo.com -m admin@foo.com --config-dir /tmp/le auth
DEBUG:root:Logging level set at 10
DEBUG:root:Discovered plugins: PluginsRegistry(PluginEntryPoint#apache,PluginEntryPoint#nginx,PluginEntryPoint#standalone)
Traceback (most recent call last):
  File "/root/lets-encrypt-preview/venv/bin/letsencrypt", line 9, in <module>
    load_entry_point('letsencrypt==0.1', 'console_scripts', 'letsencrypt')()
  File "/root/lets-encrypt-preview/letsencrypt/cli.py", line 380, in main
    return args.func(args, config, plugins)
  File "/root/lets-encrypt-preview/letsencrypt/cli.py", line 118, in auth
    config, args.authenticator, plugins)
  File "/root/lets-encrypt-preview/letsencrypt/display/ops.py", line 93, in pick_authenticator
    config, default, plugins, question, (interfaces.IAuthenticator,))
  File "/root/lets-encrypt-preview/letsencrypt/display/ops.py", line 69, in pick_plugin
    verified.prepare()
  File "/root/lets-encrypt-preview/letsencrypt/plugins/disco.py", line 187, in prepare
    return [plugin_ep.prepare() for plugin_ep in self._plugins.itervalues()]
  File "/root/lets-encrypt-preview/letsencrypt/plugins/disco.py", line 91, in prepare
    self._initialized.prepare()
  File "/root/lets-encrypt-preview/letsencrypt_apache/configurator.py", line 147, in prepare
    temp_install(self.conf('mod-ssl-conf'))
  File "/root/lets-encrypt-preview/letsencrypt_apache/configurator.py", line 1175, in temp_install
    shutil.copyfile(constants.MOD_SSL_CONF, options_ssl)
  File "/usr/lib/python2.7/shutil.py", line 83, in copyfile
    with open(dst, 'wb') as fdst:
IOError: [Errno 2] No such file or directory: '/etc/letsencrypt/options-ssl.conf'
```

**Comment by schoen** [comment]
Conceivably related to #443?  (I don't know what's behind either problem, but #443 also seemed to produce some errors related to references to `options-ssl.conf`.)

**Comment by kuba** [comment]
Fixed in #451

**Comment by kuba** [comment]
Ups, meant that it's fixed in #440.
